components/BottomAppBar/src/MDCBottomAppBarView.m

```diff
@@ -86,6 +88,8 @@ - (void)commonMDCBottomAppBarViewInit {
 
   self.barTintColor = UIColor.whiteColor;
   self.shadowColor = UIColor.blackColor;
+  self.elevation = MDCShadowElevationBottomNavigationBar;
```

Comment:
Please assign the elevation to the ivar directly. Accessing `self` in an initializer is discouraged.  In this case, `setElevation:` calls out to the category method and triggers a bunch of other code.

```objc
_elevation = MDCShadowElevationBottomNavigationBar;
```